src/Epay.php

```diff
@@ -131,9 +131,9 @@ public function receiveNotification($requestInputs)
                     $item = [
                         'invoice'  => $regs[1],
                         'status'   => $regs[2],
-                        'pay_date' => $regs[4],
-                        'stan'     => $regs[5],
-                        'bcode'    => $regs[6],
+                        'pay_date' => $regs[4] ?? '',
```

Comment:
Please replace ?? with ternary operator because we need to support PHP 5.*.
` 'pay_date' => isset($regs[4]) ? $reqs[4] : '',`
` 'stan'     =>  isset($regs[5]) ? $reqs[5] : '',`
` 'bcode'    => isset($regs[6]) ? $reqs[6] : '', `